extension.json

```diff
@@ -2,7 +2,7 @@
   "displayName": "Avo Inspector",
   "name": "avo-inspector",
   "platform": "web",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
Does this not deserve a minor bump?

Reply:
will update to 1.2.0 before releasing finally, just having it patches while developing as i cant undo anything i publish